packages/studio/src/components/UndefinedMenuButton.tsx

```diff
@@ -70,9 +70,9 @@ export default function UndefinedMenuButton({
           aria-label="Toggle undefined value menu"
         />
         {isOpen && (
-          <div className="absolute mt-3">
+          <div className="relative mt-3 left-[250px]">
```

Comment:
I checked out the branch locally, and it looks like when the menu is open, the scrolling area extends much further than what's needed to display the menu. I think that's because of the `left-[250px]` here. If we remove this, can we tweak the positioning of the `ul` below to get the desired position without the extra overflow?

Reply:
Good catch, I didn't notice that. Fixed.